src/components/organisms/OcResource/OcResource.vue

```diff
@@ -342,7 +343,8 @@ export default {
           return {
             name: "Documents",
             path: "/Documents",
-            icon: "folder",
+            icon: "resource-type-folder",
```

Comment:
Examples need to be adapted now. Should work automagically without the `icon` and `iconColor` keys in the objects.